Collection page broken

The collection page currently displays this error when running the dev server:

```
ERROR in ./components/Collections/Collection.js
Module Error (from /code/node_modules/eslint-loader/index.js):

/code/src/client/components/Collections/Collection.js
  122:11  error  'contact' is assigned a value but never used  no-unused-vars
```

I think the removal of the contact email from display means the variable can now be removed.

--- a/src/client/components/Collections/Collection.js
+++ b/src/client/components/Collections/Collection.js
@@ -119,9 +119,6 @@ export default class CollectionList extends Component {
 
     const tweetCount = this.props.search.tweetCount.toLocaleString()
 
-    const contact = this.props.search.creator.email
-      ? <a href={`mailto:${this.props.search.creator.email}`}>{this.props.search.creator.email}</a>
-      : 'No contact provided.'
     let tweets = 'Loading tweets...'
     if (this.props.search.tweets.length > 0) {
       tweets = this.props.search.tweets.slice(this.randomTweet, this.randomTweet + 2).map((t, i) => {

